stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/attachments/factory/AttachmentsPickerMediaCaptureTabFactory.kt

```diff
@@ -120,36 +119,4 @@ public class AttachmentsPickerMediaCaptureTabFactory(private val pickerMediaMode
             MissingPermissionContent(cameraPermissionState)
         }
     }
-
-    /**
-     * Returns if we need to check for the camera permission or not.
-     *
-     * @param context The context of the app.
-     * @return If the camera permission is declared in the manifest or not.
-     */
-    private fun isCameraPermissionDeclared(context: Context): Boolean {
-        return context.packageManager
-            .getPackageInfo(context.packageName, PackageManager.GET_PERMISSIONS)
-            .requestedPermissions
-            .contains(Manifest.permission.CAMERA)
-    }
-
-    /**
-     * Define which media type will be allowed.
-     */
-    public enum class PickerMediaMode {
-        PHOTO,
-        VIDEO,
-        PHOTO_AND_VIDEO,
-    }
```

Comment:
Hey @VelikovPetar 👋 

We can't delete public classes within a single PR due to compatibility reasons.
We have to deprecate this one first with `@Deprecated`  annotation:
-  **warning**  level
- **error** level
- then delete 🙂

Reply:
Addressed here: 6dea00f188f482d882fbe8e4a89f05c574c08dd9

I basically reverted this change, as I realised there wasn't really a need to extract this class.